scripts/scripts_object_storage/openstack_swift/local-setup-spark.sh

```diff
@@ -0,0 +1,10 @@
+ #!/usr/bin/env bash
+# Copyright 2016 Yahoo Inc.
+# Licensed under the terms of the Apache 2.0 license.
+# Please see LICENSE file in the project root for terms.
+#
+# This script install Spark locally
+
+wget http://archive.apache.org/dist/spark/spark-2.0.1/spark-2.0.1-bin-hadoop2.7.tgz
```

Comment:
Yes, we definitely can use http://archive.apache.org/dist/spark/spark-2.0.0/spark-2.0.0-bin-hadoop2.7.tgz.

"Why do we need hadoop-openstack as a dependency into pom.xml? In our pomx.xml, Apache Spark etc are included for compilation purpose only, and thus scope "provided". hadoop-openstack should be given in your Hadoop cluster, not in CaffeOnSpark."
  ~ I refered https://spark.apache.org/docs/1.5.1/storage-openstack-swift.html and came to a conclusion that `hadoop-openstack` dependency should be added to CaffeOnSpark as well.

Now that you asked the question, I kind of question my approach as well. 

The most important step was to get `hadoop-openstack-2.7.1.jar` file ready in `HADOOP_CLASSPATH`. Do you want me to try this again without modifying the CaffeOnSpark/caffe-grid/pom.xml file with any hadoop-openstack dependency ?

Thanks,
Arun